### ISSUE TITLE
Document files viewed as video files inside the manager

### ISSUE THREAD

**Issue by Harksa** [PiranhaCMS/piranha.core#629]
Hello, 
I've added the ability to upload some document files like .txt and .zip files inside my server. They upload perfectly but when I click on them in the manager, the movie player is popping up.

It's not too much of a problem but I just wanted to point it out.

**Comment by filipjansson** [comment]
Hi!

Have you registered the .txt and .zip as documents or as another media type?

BR

**Comment by Harksa** [comment]
I used those line of codes : 
```
App.MediaTypes.Documents.Add(".zip","application/zip");
App.MediaTypes.Documents.Add(".txt","text/plain");
```

I think it was the way to do it, am I right ?

**Comment by tidyui** [comment]
We have fixed this in `7.0` and added a new media type `Resources` where you can add content types that can't be previewed, in your case `zip` files. See attached screenshots below:

![Skärmavbild 2019-05-23 kl  14 45 30](https://user-images.githubusercontent.com/781266/58254185-7dace900-7d6a-11e9-93b1-335ee2e8136f.png)
![Skärmavbild 2019-05-23 kl  14 48 04](https://user-images.githubusercontent.com/781266/58254202-82719d00-7d6a-11e9-8529-6324492c4f43.png)
